### PR TITLE
GPUAggregator: Fix Texture/Buffer resource leaks.

### PR DESCRIPTION
**Pull request by 1chandu**
For #3039 

#### Background
GPUAggregator: Couple of resources are not properly deleted, collect and delete these resources.

#### Change List
- GPUAggregator: Fix Buffer/Texture leaks

### CODE REVIEW

File: modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js [hunk 1]
```diff
@@ -610,12 +639,7 @@ export default class GPUGridAggregator {
       isInstanced: true,
       instanceCount: 0,
       attributes: {
-        position: new Buffer(gl, {
-          data: new Float32Array([0, 0]),
-          accessor: {
-            size: 2
-          }
-        })
+        position: this.state.resources['all-aggregation-buffer']
```
Comment (Pessimistress) [inline modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js]: You should be able to use `{constant: true, value: [0, 0]}`

File: modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js [hunk 2]
```diff
@@ -152,11 +151,15 @@ export default class GPUGridAggregator {
 
     positionsBuffer && positionsBuffer.delete();
     position64Buffer && position64Buffer.delete();
-    this.deleteResources(framebuffers);
-    this.deleteResources(maxMinFramebuffers);
-    this.deleteResources(minFramebuffers);
-    this.deleteResources(maxFramebuffers);
-    this.deleteResources(meanTextures);
+    this.deleteResources([
+      framebuffers,
```
Comment (Pessimistress) [inline modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js]: Can we track some of these with `resources`?

**Comment by Pessimistress** [comment]
Can you upgrade luma.gl to 7.0.1 which includes https://github.com/uber/luma.gl/pull/1087

File: modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js (hunk 1 above)
Comment (1chandu) [reply modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js]: Yep, changed to [0, 0]

File: modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js (hunk 2 above)
Comment (1chandu) [reply modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js]: Some of these are used in several places, so keeping separate objects make it much easy to read and less error prone.

**Comment by 1chandu** [comment]
> Can you upgrade luma.gl to 7.0.1 which includes [uber/luma.gl#1087](https://github.com/uber/luma.gl/pull/1087)

@Pessimistress Upgraded to 7.0.1, no new issues found in tests.

**Pull request merged**